### PR TITLE
LuckyFlow::Element#value uses property instead of attribute

### LINKED ISSUE #86: Get the value of an element with property rather than attribute
`LuckyFlow::Element#value` is currently calling `attribute("value")` to get the value of the element. Source:
https://github.com/luckyframework/lucky_flow/blob/0afe110f00ff9edaa21300162516d27e79289a24/src/lucky_flow/element.cr#L16-L18

While this works with Chrome, it will not work if Firefox or any other browser driver is added. This is because `attribute` is meant to reference the static fields of an element. To get a dynamic field like value, we should be calling `property` instead. Reference this stackoverflow question for more information on the difference between attribute and property: https://stackoverflow.com/questions/6003819/what-is-the-difference-between-properties-and-attributes-in-html

It seems to be up to the browser whether or not attribute is updated for dynamic properties like value. Chrome is the only one I've seen so far to update it, but Safari and Firefox definitely do not.

The fix is simple and should "just work" :

```crystal
def value
  property("value")
end
```

### PR DESCRIPTION
**Pull request by matthewmcgarvey**
The reasoning behind this was laid out here: https://github.com/luckyframework/lucky_flow/issues/86

TLDR: switching to `property` allows us to add support for other browsers in the future

Fixes #86

### CODE REVIEW

File: src/lucky_flow/element.cr [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class LuckyFlow::Element
   private getter raw_selector
   getter inner_text
-  delegate text, click, send_keys, displayed?, attribute, to: element
+  delegate text, click, send_keys, displayed?, attribute, property, to: element
```
Comment (matthewmcgarvey) [inline src/lucky_flow/element.cr:4]: while `attribute` is no longer used in the class, I wasn't sure about removing it from this list since it is technically public API. I can remove it, though

Comment (jwoertink) [reply src/lucky_flow/element.cr:4]: I think we should keep it in. At least for this next release. We can see about deprecating it later.

**Comment by matthewmcgarvey** [comment]
@jwoertink Do you mean a spec testing the `value` method? If so, here it is https://github.com/luckyframework/lucky_flow/blob/78f895d27503f68b730797cac711da9919cea6d7/spec/lucky_flow_spec.cr#L77-L90

**Comment by jwoertink** [comment]
I was thinking more on the `property` method itself, but now I think value being called is probably good enough. 👍
